src/App.js

```diff
@@ -20,14 +22,45 @@ const createRoute = (path, component, style={marginTop: "2em"}) => {
     }
 }
 
+const hrOfferButtons = [
+    {
+        text: "Edit",
+        action: (offer, history) => {
+            history.push(`/hr/offers/edit/${offer.id}`)
+        }
+    },
+    {
+        text: "Delete",
+        action: (offer, history) => {
+            withSwal({
+                loadingTitle: "Deleting offer",
+                promise: () => offersAPI.deleteOffer(offer.id),
+                successSwalTitle: "Successfully deleted",
+                successSwalText: "Offer has been deleted",
+                confirmButtonText: "Ok",
+                successFunction: () => history.go(0),
+                errorSwalTitle: "Offer not deleted",
+                errorSwalText: "We couldn't delete your offer",
+                errorConfirmButtonText: "Ok"
+            })
+        },
+        style: {
+            color: "secondary"
+        }
+    }
+
+]
+
 const routes = [
     createRoute("/", <div><LandingPage /><a href='https://www.freepik.com/vectors/people' style={{display:"none"}}>People vector created by pikisuperstar - www.freepik.com</a></div>),
     createRoute("/interview/:interviewId/:companyId", <Meeting />, {}),
     createRoute("/interview/:interviewId/", <Meeting />, {}),
-    createRoute("/offers/add", <OfferForm />),
+    createRoute("/offers/add", <OfferForm onSubmit={(form) => offersAPI.create(form)}/>),
```

Comment:
Zmień tu route'a na "/hr/offers/add", skoro i tak tylko HR może mieć możliwość dodawania oferty. Jak już zmienisz, to przenieś obok hrów (linijka 62)